.github/workflows/android.yml

```diff
@@ -103,6 +103,13 @@ jobs:
       id-token: write
       contents: read
     uses: pytorch/test-infra/.github/workflows/mobile_job.yml@main
+    strategy:
+      matrix:
+          # https://github.com/pytorch/executorch/blob/main/examples/demo-apps/android/LlamaDemo/README.md#alternative-2-build-from-local-machine
+          # mentions that tiktoken is only for Llama3. So, we can export it later in another archive
+          # like https://ossci-assets.s3.amazonaws.com/executorch-android-llama2-7b.zip when this is
+          # updated to run Llama3
+          tiktoken: [OFF]
```

Comment:
I think the reason of https://github.com/pytorch/executorch/pull/4203 updating this to include tiktoken tokenizer is to ensure the demo app can build for Llama3. So as mentioned that this job should focus on validating different flavors of android demo app builds. If we remove tiktoken here, there will be a tcc regression.

Reply:
Yeah, this covers only llama2 without tiktoken atm.  To cover llama3, for example, we would need to zip the exported model and the tokenizer it uses and upload the archive onto S3, i.e. https://github.com/pytorch/executorch/blob/main/.github/workflows/android.yml#L122.  Then the archive can be passed to the job as its `extra-data` parameter to be copied to the device.